### PR TITLE
Feat: add rebuild amplify sites script

### PR DESCRIPTION
**Pull request by alexanderleegs**

### CODE REVIEW

File: rebuild-sites-amplify/index.js
Comment (seaerchin) [inline rebuild-sites-amplify/index.js:1]: add readme

File: rebuild-sites-amplify/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const MAX_RESULTS_PER_PAGE = 100
+
+async function rebuildAllAmplifySites() {
+  let sitesList = {}
+  let nextToken
+  const listCommand = new ListAppsCommand({
+    maxResults: MAX_RESULTS_PER_PAGE,
+  })
+  response = await (awsClient.send(listCommand))
```
Comment (seaerchin) [inline rebuild-sites-amplify/index.js:23]: are we not missing a declaration here

**Review by seaerchin — APPROVED** [review]

File: rebuild-sites-amplify/index.js
Comment (alexanderleegs) [reply rebuild-sites-amplify/index.js:1]: Done upstream!

File: rebuild-sites-amplify/index.js (hunk 1 above)
Comment (alexanderleegs) [reply rebuild-sites-amplify/index.js]: https://github.com/isomerpages/isomer-tooling/pull/43/commits/25e251eee1d15d34679752e2341314172fa9b0c1

**Pull request merged**